src/settings.ts

```diff
@@ -89,6 +89,9 @@ export const DUST_STORM_TEXT_2 =
 
 // How often to refresh events schedule
 export const REFETCH_SCHEDULE_MS = 10 * 60 * 1000; // 10 mins
+export const SHORT_EVENT_LENGTH = 10;
+export const MIN_LONG_EVENT_LENGTH = 15;
+export const LONG_EVENT_LENGTH = 60;
```

Comment:
Seemingly `LONG_EVENT_LENGTH` is specified in minutes. So presumably they all are?

Something like this perhaps?
```suggestion
// How often to refresh events schedule
export const REFETCH_SCHEDULE_MS = 10 * 60 * 1000; // 10 mins
export const SCHEDULE_SHORT_EVENT_LENGTH_MIN = 10; // minutes
export const SCHEDULE_MIN_LONG_EVENT_LENGTH_MIN = 15; // minutes
export const SCHEDULE_LONG_EVENT_LENGTH_MIN = 60; // minutes
```

Reply:
What's the difference between `MIN_LONG_EVENT_LENGTH` and `LONG_EVENT_LENGTH`?

And why do we need both of `SHORT_EVENT_LENGTH` and `MIN_LONG_EVENT_LENGTH`?

Presumably if an event isn't short, it would be long, no?

Reply:
`SCHEDULE_MIN_LONG_EVENT_LENGTH_MIN = 15` is a minimal length/width of an event to display any ui controls on it ( expand/bookmark w/e ) otherwise there's just no space to display those
i've changed naming in 9b0919c, but if you come up with a better one, please do suggest it